cfgov/unprocessed/css/organisms/post-preview.less

```diff
@@ -102,20 +131,50 @@
                 vertical-align: bottom;
             }
         } );
+
+        &_left,
+        &_right {
+            display: inline-block;
+            float: none;
```

Comment:
This float is overriding https://github.com/cfpb/capital-framework/blob/canary/src/cf-typography/src/molecules/meta-header.less#L9-L15
Do we know if this is used anywhere else or could the float be removed entirely from CF? If it could can we put a TODO here and/or create an issue in CF?

Reply:
I agree that the `TODO` or issue would be helpful here.

Reply:
I will create an issue about this `float`!